flag.go

```diff
@@ -578,8 +578,16 @@ func withEnvHint(envVar, str string) string {
 	return str + envText
 }
 
-func stringifyFlag(f Flag) string {
+func flagValue(f Flag) reflect.Value {
 	fv := reflect.ValueOf(f)
+	for fv.Kind() == reflect.Ptr {
+		fv = reflect.Indirect(fv)
+	}
+	return fv
```

Comment:
Ack!  Pointers to pointers aren't covered, AFAICT, although that's getting pretty edgy.  Maybe compare to the zero value instead?  (ref https://golang.org/src/reflect/value.go#L2075)